gnmi/gnmit/gnmit.go

```diff
@@ -58,10 +58,14 @@ const (
 )
 
 // GNMIServer implements the gNMI server interface.
-type GNMIServer struct {
+type GNMIServer[T ygot.GoStruct] struct {
```

Comment:
do we really need to support different schemas? could this just use the oc.Schema